node-notes-cli/app-copy.js

```diff
@@ -0,0 +1,75 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+
+const [, , operation, argv3, argv4] = process.argv;
+app(operation, argv3, argv4);
+
+async function app(operation, argv3, argv4) {
+  switch (operation){
+    case 'read':
+       read();
+      break;
+    case 'create':
+      create(argv3);
+      break;
+    case 'delete':
+      deleteId(argv3);
+      break;
+    case 'update':
+      update(argv3, argv4);
+      break;
+    default:
+      console.log('Invalid operation, only read/update/create/delete are accepted');
+  }
+}
+
+async function read() {
+  try {
+    const data = JSON.parse(await readFile('data.json'));
```

Comment:
You repeat this line of code several times. To make it easier to maintain, you should extract it to a separate function.